charts/dvpe-deployment-gloo/templates/gloo_authConfigPlugin.yaml

```diff
@@ -39,20 +39,22 @@ spec:
               {{- if .Values.gloo.authConfig.spec.configs.oauth.cookieDomain }}
               domain: {{ .Values.gloo.authConfig.spec.configs.oauth.cookieDomain | quote }}
               {{- end }}
-              maxAge: 0
+              maxAge: {{ .Values.gloo.authConfig.spec.configs.oauth.maxAge }}
+            {{- if .Values.gloo.authConfig.spec.configs.oauth.cache.enabled }}
+            redis:
+              cookieName: {{ .Values.gloo.authConfig.spec.configs.oauth.cache.cookieName }}
+              options:
+                host: {{ .Values.gloo.authConfig.spec.configs.oauth.cache.host }}
+            {{- end }}
     {{- end }}
     {{- if .Values.gloo.authConfig.spec.configs.tokenValidationPlugin.enabled }}
```

Comment:
not for now, but I would prefer to have several sub-charts for each plugin in a later version.